sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
-
+## 1.0.0-beta.4 (2021-05-12)
+#### New Features
+* Todo
```

Comment:
Add or remove

Reply:
Yes, we will have one more release before public preview next week, will update the changelog accordingly.